static/build.yml

```diff
@@ -11,3 +11,7 @@ dependencies:
     - -f https://download.pytorch.org/whl/torch_stable.html
     - tensorflow==2.3.1
     - tensorflow-probability==0.11.1
+    - paddlepaddle-gpu==2.2.2
+    - -i https://mirror.baidu.com/pypi/simple
+    - paddlepaddle==2.2.2 
```

Comment:
Can we remove `paddlepaddle` since we have `paddlepaddle-gpu`

Reply:
I tested locally and I have to install both to make it work with CUDA 10.2.  I am not sure what will happen on the server.